sigpro/sigpro_pipeline_demo.ipynb

```diff
@@ -0,0 +1,485 @@
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>turbine_id</th>\n",
+       "      <th>signal_id</th>\n",
+       "      <th>xvalues</th>\n",
+       "      <th>yvalues</th>\n",
+       "      <th>sampling_frequency</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>T001</td>\n",
+       "      <td>Sensor1_signal1</td>\n",
+       "      <td>2020-01-01 00:00:00</td>\n",
+       "      <td>[0.43616983763682876, -0.17662312586241055, 0....</td>\n",
+       "      <td>1000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>T001</td>\n",
+       "      <td>Sensor1_signal1</td>\n",
+       "      <td>2020-01-01 01:00:00</td>\n",
+       "      <td>[0.8023828754411122, -0.14122063493312714, -0....</td>\n",
+       "      <td>1000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>T001</td>\n",
+       "      <td>Sensor1_signal1</td>\n",
+       "      <td>2020-01-01 02:00:00</td>\n",
+       "      <td>[-1.3143142430046044, -1.1055740033788437, -0....</td>\n",
+       "      <td>1000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>T001</td>\n",
+       "      <td>Sensor1_signal1</td>\n",
+       "      <td>2020-01-01 03:00:00</td>\n",
+       "      <td>[-0.45981995520032104, -0.3255426061995603, -0...</td>\n",
+       "      <td>1000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>T001</td>\n",
+       "      <td>Sensor1_signal1</td>\n",
+       "      <td>2020-01-01 04:00:00</td>\n",
+       "      <td>[-0.6380405111460377, -0.11924167777027689, 0....</td>\n",
+       "      <td>1000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  turbine_id        signal_id             xvalues  \\\n",
+       "0       T001  Sensor1_signal1 2020-01-01 00:00:00   \n",
+       "1       T001  Sensor1_signal1 2020-01-01 01:00:00   \n",
+       "2       T001  Sensor1_signal1 2020-01-01 02:00:00   \n",
+       "3       T001  Sensor1_signal1 2020-01-01 03:00:00   \n",
+       "4       T001  Sensor1_signal1 2020-01-01 04:00:00   \n",
+       "\n",
+       "                                             yvalues  sampling_frequency  \n",
+       "0  [0.43616983763682876, -0.17662312586241055, 0....                1000  \n",
+       "1  [0.8023828754411122, -0.14122063493312714, -0....                1000  \n",
+       "2  [-1.3143142430046044, -1.1055740033788437, -0....                1000  \n",
+       "3  [-0.45981995520032104, -0.3255426061995603, -0...                1000  \n",
+       "4  [-0.6380405111460377, -0.11924167777027689, 0....                1000  "
```

Comment:
based on our previous discussions, is this the new supported format where we have xvalues as a column of type list (like yvalues)? Or is it the original format?

1. If it is the former, I would make `xvalues` a list just like yvalues.
2. If it is the latter, I would change the column name to `timestamp` to follow the dummy data https://raw.githubusercontent.com/sintel-dev/SigPro/master/sigpro/data/demo_timeseries.csv

Reply:
The xvalues column has been revised to be a list, just like yvalues -- should be correct now.